### PR TITLE
Some minor revisions for clarity

### PR DESCRIPTION
**Pull request by rebeksburnett**
I made a lot of small changes. But then, I also rewrote a couple sentences. Please see what you think.

### CODE REVIEW

**Review by cloveras — APPROVED** [review]
It's .. MAGNIFICENT!

File: README.md [hunk 1]
```diff
@@ -115,30 +115,30 @@ The partner only needs access to the test environment to complete an integration
...
 When the integration checklist is completed, notify Vipps Integration
-[integration@vipps.no](integration@vipps.no)
-as described in the checklist with `orderId` examples from the test environment,
-pilot customer info and description of the implemented solution.
+([integration@vipps.no](integration@vipps.no))
+as described in the checklist, with `orderId` examples from the test environment,
+pilot customer info, and a description of the implemented solution.
 
-Vipps Integration will verify the integration, and contact the partner.
+Vipps Integration will verify the integration and take contact with you.
```
Comment (rebeksburnett) [inline README.md:137]: ok - something to think about:
We have used "you" in the text above and I think it's much more readable for the partner to continue with this here.
But, if we need this to be used by people who are not partners, then we should probably rewrite all the "you" to the more format, "the partner".

File: README.md [hunk 2]
```diff
@@ -156,18 +156,18 @@ for information about deprecation notices, etc.
 ## Partner keys
 
 As a partner, you manage transactions on behalf of Vipps merchants.
-Vipps provides you with _partner keys_, which allows you to use your own API credentials to
-make API calls on behalf of your merchants (only for the sales units that are linked to you as a partner, though).
-Please note that partner keys can not be used for Vipps Login, here you need to use the merchant's own keys.
+Vipps provides you with _partner keys_, which allow you to use your own API credentials to
+make API calls on behalf of your merchants (i.e., the sales units that are linked to you as a partner).
+
+**_NOTE:_**  Partner keys cannot be used for Vipps Login. You need to use the merchant's own keys in the login API.
```
Comment (rebeksburnett) [inline README.md:162]: Subtle change... pls check

File: README.md [hunk 3]
```diff
@@ -219,28 +216,19 @@ making payments, refunds, etc) on behalf of another merchant.
...
+  [test environment](https://github.com/vippsas/vipps-developers/blob/master/vipps-test-environment.md),
+  you must merchant API keys. If you are not a Vipps merchant and do not have these keys, you will need to use the merchant keys belonging to one of your merchants.
+* Vipps can not send the merchant's API keys to you. You must get them from the merchant in a secure way (if partner keys are not used).
   See:
   [Getting started: Get credentials](https://github.com/vippsas/vipps-developers/blob/master/vipps-getting-started.md#get-credentials)
   for more details.
-* If the merchant is unable to provide the API keys to the partner in a secure
-  way, the merchant _can_ create a user for the partner,
-  [as described in detail with screenshots here](add-portal-user.md):
-  1. Log in with BankID on portal.vipps.no
```
Comment (rebeksburnett) [inline README.md:237]: Here, I removed the steps as I think it's better if they do go straight to the link with the screenshots. Then, we don't need to maintain this in 2 places and it makes it shorter...

**Pull request merged**